### PR TITLE
Migrate ci to git hub actions

### PR DESCRIPTION
**Pull request by gotanygrapes-lemonade**
### Context

We have [a sensible default](https://docs.envato.net/architecture-guild/sensible-defaults/expected/open-source-ci.html) to **not** use our internal Buildkite agents to run CI for open-source projects.

However, we're doing that for a number of open-source Buildkite plugins. (Probably historical, before the default was in-place.) Let's update these projects to use GitHub Actions.

### Considerations

An [example GitHub Actions configuration](https://github.com/envato/heroku-container-deploy-buildkite-plugin/blob/16f73f647bbb6e5cca4217ec5413e6e3d3bb2f41/.github/workflows/test.yml) we can use for inspiration (or copy).

- Adding a YAML file to the `.github/workflows/` directory should be enough to enable GitHub actions
- Delete Buildkite pipelines by navigating to their settings page in a browser and click the `Delete Pipeline` in the pipeline management section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ECS Scheduled Tasks Buildkite Plugin
 
-A [Buildkite plugin](https://buildkite.com/docs/agent/v3/plugins) for deploying [Amazon ECS Scheduled Tasks](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/scheduling_tasks.html).
```
Comment (orien) [inline README.md:3]: Let's not delete this line.

**Review by orien — APPROVED** [review]
👍  The GitHub Actions build is failing, but I see it was failing on Buildkite as well.

Let's delete the `.buildkite` directory also.

**Review by runlevel5 — APPROVED** [review]
LGTM

P/S: Please get all failed tests addressed

**Review by lucasrsantos1 — APPROVED** [review]

File: tests/command.bats [hunk 2]
```diff
@@ -2,19 +2,19 @@
 
 load '/usr/local/lib/bats/load.bash'
 
-# Uncomment to enable stub debug output:
+# Uncgomment to enable stub debug output:
```
Comment (orien) [inline tests/command.bats:5]: This looks unintentional.

**Review by orien — APPROVED** [review]
Nice one.

File: tests/command.bats (hunk 2 above)
Comment (Bing-Violet) [reply tests/command.bats:5]: oops, will fix it. Thanks Orien.

**Pull request merged**